src/ensembl/src/shared/components/selected-species/SelectedSpecies.scss

```diff
@@ -9,6 +9,8 @@
   text-overflow: ellipsis;
   white-space: nowrap;
   color: $white;
+  float: left;
+  margin-bottom: 10px;
```

Comment:
1) SelectedSpecies itself shouldn't have a margin bottom. Remember how it went for ExternalReference?
2) I'm sure we can find something better than floats! What if you add `display: flex; flex-wrap: wrap` to parent?

Reply:
> What if you add display: flex; flex-wrap: wrap to parent? 

That worked. Thanks